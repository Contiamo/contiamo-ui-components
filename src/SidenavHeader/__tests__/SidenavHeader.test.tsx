import * as React from "react"

import { render, shallow } from "enzyme"
import { SidenavHeader as ThemelessSideNavHeader } from "../../"

import wrapDefaultTheme from "../../utils/wrap-default-theme"

const SidenavHeader = wrapDefaultTheme(ThemelessSideNavHeader)

describe("SidenavHeader Component", () => {
  it("Should initialize properly", () => {
    expect(render(<SidenavHeader label="Chapter One" to="/one" />)).toMatchSnapshot()
  })

  it("Expect to have a valid wrapper after loaded", () => {
    const wrapper = shallow(<SidenavHeader label="Chapter One" to="/one" />)
    expect(wrapper).toBeTruthy()
  })
})
