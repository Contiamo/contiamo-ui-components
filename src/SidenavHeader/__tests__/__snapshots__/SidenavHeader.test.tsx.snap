// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SidenavHeader Component Should initialize properly 1`] = `
<div
  class="css-ftf61z"
>
  <div
    class="css-p29iif"
  />
  <a
    class="css-synphd-sidenavheader"
    href="/one"
    label="Chapter One"
  >
    <div
      class="css-11wld9j"
    >
      <div
        class="css-1ibxbsn"
      >
        Chapter One
      </div>
      <div
        class="css-w9ofnk"
      />
    </div>
  </a>
</div>
`;
